core/src/main/scala/cats/Cartesian.scala

```diff
@@ -14,6 +14,12 @@ import simulacrum.typeclass
  */
 @typeclass trait Cartesian[F[_]] {
   def product[A, B](fa: F[A], fb: F[B]): F[(A, B)]
+
+  @inline final def *>[A, B](fa: F[A])(fb: F[B])(implicit F: Functor[F]): F[B] =
+    F.map(product(fa, fb)) { case (_, b) => b }
```

Comment:
I don’t like this. If you have a Functor for Cartesian you have an Apply, no?

I’m -1 in accepting other type classes on F in methods for a typeclass of F in most cases.

Reply:
Also, I don’t see why these aren’t exactly symbols for the methods you moved to Apply.

Reply:
I'll add these as aliases for those in apply :)